scripts/gen_psql_tests

```diff
@@ -189,6 +189,7 @@ commands = set([
     "SET TRANSACTION",
     "SHOW",
     "START TRANSACTION",
+    "TABLE",
```

Comment:
Thank you! Great find. The test generator is "dumb" so that it doesn't know the TABLE statement is really considered a SELECT statement. I knew it wasn't very smart at the time I wrote it, but it's good enough for now. Still, I think the grammar definition itself should consider TABLE as a type of SELECT statement even if the tests aren't quite consistent yet with that taxonomy. 

Perhaps we can add an issue or TODO somewhere to make this test generator smart enough to know that TABLE statements are considered SELECT statements? It does not need to be done now.

Reply:
The idea and realization of test generator is well enough besides the parsing of "create function" statement.
There a lot of semicolons inside functions. I see big challenge here.

Also need think about `root` because of using semicolon.
```
root
    : stmt ((SEMI stmt)+ SEMI)? EOF
    ;
```
https://github.com/postgres/postgres/blob/master/src/test/regress/sql/plpgsql.sql